TLM/TLM/Patch/_VehicleAI/_PassengerCarAI/GetLocalizedStatusPatch.cs

```diff
@@ -0,0 +1,87 @@
+        public delegate ushort GetDriverInstance(ushort vehicleID, ref Vehicle vehicleData);
+
+        [UsedImplicitly]
+        public static bool Prefix(ref string __result,
+                                   GetDriverInstance getDriverInstance,
+                                   // Harmony magic END
+                                   ushort vehicleID,
+                                   ref Vehicle data,
+                                   out InstanceID target) {
+            CitizenManager citizenManager = CitizenManager.instance;
+            ushort driverInstanceId = getDriverInstance(vehicleID, ref data);
+            ushort targetBuildingId = 0;
+            bool targetIsNode = false;
+
+            if (driverInstanceId != 0) {
+                if ((data.m_flags & Vehicle.Flags.Parking) != 0) {
+                    uint citizen = citizenManager.m_instances.m_buffer[driverInstanceId].m_citizen;
+                    if (citizen != 0u
+                        && citizenManager.m_citizens.m_buffer[citizen].m_parkedVehicle != 0) {
+                        target = InstanceID.Empty;
+                        __result = Locale.Get("VEHICLE_STATUS_PARKING");
+                        return false;
+                    }
+                }
+
+                targetBuildingId = citizenManager.m_instances.m_buffer[driverInstanceId]
+                                                 .m_targetBuilding;
+                targetIsNode = (citizenManager.m_instances.m_buffer[driverInstanceId].m_flags
+                                & CitizenInstance.Flags.TargetIsNode) != CitizenInstance.Flags.None;
+            }
+
+            if (targetBuildingId == 0) {
+                target = InstanceID.Empty;
+                __result = Locale.Get("VEHICLE_STATUS_CONFUSED");
+                return false;
+            }
+
+            string ret;
+            bool leavingCity =
+                (BuildingManager.instance.m_buildings.m_buffer[targetBuildingId].m_flags
+                 & Building.Flags.IncomingOutgoing) != Building.Flags.None;
+            if (leavingCity) {
+                target = InstanceID.Empty;
+                ret = Locale.Get("VEHICLE_STATUS_LEAVING");
+            } else {
+                target = InstanceID.Empty;
+                if (targetIsNode) {
+                    target.NetNode = targetBuildingId;
+                } else {
+                    target.Building = targetBuildingId;
+                }
+
+                ret = Locale.Get("VEHICLE_STATUS_GOINGTO");
+            }
+
+            // NON-STOCK CODE START
+            if (Options.parkingAI) {
+                ret = AdvancedParkingManager.Instance.EnrichLocalizedCarStatus(
+                    ret,
+                    ref ExtCitizenInstanceManager.Instance.ExtInstances[driverInstanceId]);
+            }
+
+            // NON-STOCK CODE END
+            __result = ret;
+            return false;
+        }
```

Comment:
If the comments about `// NON-STOCK CODE` is true, this can easily be a postfix.

Reply:
its actually more difficult to have this as a prefix that returns false!

Reply:
I can see you other `GetLocalisedStatusPatch`es are also using prefix so it would be nice to make it consistent.

Reply:
Hmm, I'll try postfix'ing that 🤔

Reply:
ok... it won't be easy since we need driver instance... It's expensive to call, so I don't think it would be good idea for double call 😕 
Transpiler will be too complex, because vanilla code is set of `if` returns (aka. _labels hell_)...
I'll leave it for now. If I'll get bored, I'll try to improve it 😅